provisioning/device/src/Utilities/Logging.cs

```diff
@@ -4,13 +4,15 @@
 using System;
 using System.Collections;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Diagnostics.Tracing;
 using System.Globalization;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.Azure.Devices.Provisioning.Client
 {
+    [ExcludeFromCodeCoverage]
```

Comment:
👍🏽

Reply:
We should be really careful about excluding things from cc. I'd like to see it called out explicitly and a case made in a code comment about why it is not needed.

We talked about one in an earlier PR for a logging class, which I agree we can omit from cc, so I'm fine with this one too.

Still needs a comment. I'd suggest we cover exclusions in stand up with a parking lot convo.